src/tools/r2rdump/x86/GcSlotTable.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Xml.Serialization;
+
+namespace R2RDump.x86
+{
+    public class GcSlotTable
+    {
+        public struct GcSlot
```

Comment:
@nattress I missed this the first time, but one of the properties in the GcSlot struct is a string. Since this isn't a value type, would it be better to make this GcSlot struct (and amd64 one) a class instead?